apps/pie-docs/src/_data/metadata.js

```diff
@@ -3,4 +3,6 @@ module.exports = {
     url: 'https://www.pie.design/',
     language: 'en',
     description: 'PIE is the global design system for Just Eat Takeaway.com, powering all our products.',
+    lastUpdated: new Intl.DateTimeFormat('en-GB').format(),
```

Comment:
I'm guessing this isn't unique per page?

Reply:
we opted for a general site last updated approach for now @xander-marjoram 👍